src/kubernetes/.lint.yml

```diff
@@ -0,0 +1,2 @@
+rules:
+  line-length: disable
```

Comment:
how is this used?  Could we add this to the CI somehow?

Reply:
currently locally with:
`yamllint -c src/kubernetes/.lint.yml src/kubernetes/deployment.template.yaml`

Reply:
ok so it's an external tool? Maybe we could add it to the docker image that kubectl/az cli run in, and run it before the `kubectl apply`.  Only if it adds value though - it's a bit late to run it at this point.

Don't really want to get into having to add stuff to the agent VM and all that :-)

Reply:
should just be able to run it in an image, it should run in the test section i think well before this bit if the flag is enabled. but we can probably leave that for this PR